src/utils.js

```diff
@@ -231,6 +231,12 @@ export function categoryIDToDBTotalsColumn(categoryId) {
   return categoryIdParts.prefix + "0001";
 }
 
+export function dbColumnToCategoryId(dbColumn) {
```

Comment:
is this still needed? The codes should be correct now, I would have thought

Reply:
yes, because the legacy census data service is still populating the map, but the metadata codes being passed around are now coming from the API.  So I needed to make a function to convert them to back to the old format